lib/reporters/console.js

```diff
@@ -143,8 +144,8 @@ internals.Reporter.prototype.end = function (notebook) {
             if (test.err.actual !== undefined &&
                 test.err.expected !== undefined) {
 
-                var actual = JSON.stringify(test.err.actual, null, 2);
-                var expected = JSON.stringify(test.err.expected, null, 2);
+                var actual = StringifySafe(test.err.actual, null, 2);
```

Comment:
@geek stringify safe internally uses JSON.stringify with a custom replacer, not sure it's worth doing part of the job twice (worst case scenario), anyway would you mind updating your benchmarks with the newer version ? Note that it can still be hugely improved (see my whining here about it https://github.com/isaacs/json-stringify-safe/pull/12).